deploy/modules/aliyun/kubernetes-monitor/main.tf

```diff
@@ -0,0 +1,38 @@
+data "template_file" "kubeconfig_filename" {
+  template = var.kubeconfig_file
+  vars = {
+    kubernetes_depedency = alicloud_cs_managed_kubernetes.k8s.client_cert
+  }
+}
+resource "null_resource" "setup-env" {
+  depends_on = [data.template_file.kubeconfig_filename]
+
+  provisioner "local-exec" {
+    working_dir = path.cwd
+    command     = <<EOS
+mkdir monitor
+echo "${local_file.kubeconfig.sensitive_content}" > monitor/config.yaml
+# Kubernetes cluster monitor
+if var.install_prometheus_operator; then
```

Comment:
this looks like it should be template inserted. All bash should have `set -euo pipefail` so errors are not ignored.